testsuite/tests/typing-unique/overwriting.ml

```diff
@@ -869,7 +892,10 @@ let mutating_tag_seq_parent r =
 Line 5, characters 15-20:
 5 |     overwrite_ r.m.x with OptionA s
                    ^^^^^
-Error: This value is "aliased" but is expected to be "unique".
+Error: This value is "aliased"
+       because it is the field "x" of the record at Line 5, characters 15-18
```

Comment:
This error message is probably a bit superfluous - of course it's the field `x` of the record.

Reply:
I think that's fine.